python/artm/model.py

```diff
@@ -638,8 +640,11 @@ def get_phi(self, topic_names=None, class_ids=None):
         if not self._initialized:
             raise RuntimeError('Model does not exist yet. Use ARTM.initialize()/ARTM.fit_*()')
 
-        phi_info = self.master.get_phi_info(model=self.model_pwt)
-        nd_array = self.master.get_phi_matrix(model=self.model_pwt,
+        if model_name is None:
+            model_name = self.model_pwt
```

Comment:
```
if model_name is None:
  model_name = self.model_pwt
```
I'm not sure if in python this works as you expect it... From what I understand this this code will actually change the default value for `model_name` argument that will affect next calls to `get_phi`. @MelLain , am I correct? How would you suggest to change this?